### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by gaetan-cordonnier**

### CODE REVIEW

**Review by Guy-deMessimy — DISMISSED** [review]

**Review by AmjadLille — DISMISSED** [review]

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:5050
```
Comment (LoicBrassart) [inline .env]: `.env` on Github ! :scream:

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^12.5.0",
     "axios": "^0.21.1",
     "emailjs-com": "^2.6.4",
+    "i": "^0.3.6",
```
Comment (LoicBrassart) [inline package.json]: What's this package ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- "Minor fix" -> 79 files -> /me :cry: 
- I think you didn't apply Prettier, most changes are cosmetic and there are some incoherences
- Some errors to fix (see comments)

File: package.json (hunk 2 above)
Comment (AmjadLille) [reply package.json]: deleted

File: .env (hunk 1 above)
Comment (ThomasDruart) [reply .env]: now deleted

**Review by AmjadLille — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**